keps/sig-autoscaling/5325-hpa-pod-selection-accuracy/kep.yaml

```
@@ -0,0 +1,45 @@
+title: KEP Template
+kep-number: 5325
+authors:
+  - "@omerap12"
+  - "@adrianmoisey"
+owning-sig: sig-autoscaling
+participating-sigs: []
+status: provisional #|implementable|implemented|deferred|rejected|withdrawn|replaced
+creation-date: 2025-05-21
+reviewers:
+  - "@soltysh"
+approvers:
+  - "@soltysh"
+
+see-also: []
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+# If the purpose of this KEP is to deprecate a user-visible feature
+# and a Deprecated feature gates are added, they should be deprecated|disabled|removed.
+stage: alpha #|beta|stable
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.34"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.34"
+  beta: "v1.35"
+  stable: "v1.36"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: HPASelectionStrategy
+    components:
+      - kube-apiserver
+      - kube-controller-manager
+disable-supported: true
+
+# The following PRR answers are required at beta release
+metrics:
+  - my_feature_metric
```

```suggestion
metrics: []
```

How necessary is it to make a metric? Taking a look at other similar features, I found an example that added a metric that would be the count of the number of times that the new API field is used. Is that necessary?

(I guess I'm asking @soltysh that question)